README.rst

```diff
@@ -13,9 +13,11 @@ aws-encryption-sdk-cli
 .. image:: https://travis-ci.org/awslabs/aws-encryption-sdk-cli.svg?branch=master
    :target: https://travis-ci.org/awslabs/aws-encryption-sdk-cli
 
```

Comment:
Looks like all the CRLF errors were fixed, but there's still a lot of trailing whitespace and some broken links: https://travis-ci.org/awslabs/aws-encryption-sdk-cli/jobs/312173892#L479

Reply:
fixed